pkg/formulaexec/formula_exec.go

```diff
@@ -48,6 +48,20 @@ type RioOutput struct {
 	Result RioResult `json:"result"`
 }
 
+// DefaultRunPathPrefix will be the prefix used to create a temporary execution directory
+const DefaultRunPathPrefix = "warpforge-run-"
+
+type ExecConfig interface {
```

Comment:
Why'd this end up needing to declare a new interface?  Can't we just concretely import the new config package?